.github/release-drafter.yml

```diff
@@ -0,0 +1,26 @@
+categories:
+  - title: 'Major Changes'
+    labels:
+      - 'major'  # c6476b
+  - title: 'Minor Changes'
+    labels:
+      - 'feature'  # 006b75
+      - 'enhancement'  # ededed
+      - 'performance'  # 555555
+      - 'docs'  # 4071a5
+  - title: 'Bugfixes'
+    labels:
+      - 'fix'
+      - 'bugfix'
+      - 'bug'  # fbca04
```

Comment:
I question if we really need all three of these? I would size `fix` would suffice.